src/styles/utils/_mixins.scss

```diff
@@ -1,6 +1,88 @@
+@import "variables";
+
 @mixin hover($_property, $_toValue) {
-  transition: #{$_property} 0.3s;
-  &:hover {
-    #{$_property}: $_toValue;
+  @media (pointer: fine) {
+    transition: #{$_property} $animation-duration;
+    &:hover {
+      #{$_property}: $_toValue;
+    }
   }
 }
+
+@mixin increase-hover-scale {
+  @include hover(transform, scale(1.05));
+}
+
+@mixin for-tablet {
+  @media (min-width: $tablet-width) {
+    @content;
+  }
+}
+
+@mixin for-desktop {
+  @media (min-width: $desktop-width) {
+    @content;
+  }
+}
+
+@mixin not-for-desktop {
+  @media (max-width: calc(#{$desktop-width} - 1px)) {
+    @content;
+  }
+}
```

Comment:
If you are not using this mixin you can remove it

Reply:
I deleted this mixin.